context-js/tests/parse_and_validate.test.ts

```diff
@@ -68,6 +68,8 @@ describe("context-js", () => {
         <testsuite name="my-test-suite" tests="1" disabled="0" errors="0" failures="1" timestamp="${validTimestamp}">
           <testcase name="failure-case" file="test.py" classname="MyClass" timestamp="${validTimestamp}" time="1">
             <failure/>
+            <system-out/>
```

Comment:
Worth leaving these in here just to know that we _can_ parse these?

Reply:
Oh lol I misread the diff and thought this PR removed them.. my b!!!!!!!!!!!!!!!!!!!!

Reply:
I'd prefer that these be added into the Rust tests since the bindings tests shouldn't be used to test core functionality since it's 1 degree removed. For example, I could hack in a fix for JS, but not have it work in Rust. If that seems like too much work, let's split a ticket for it.

Reply:
+1 to Dylan. Bindings tests should generally be an auxiliary layer to make sure we can consume things correctly. Direct functionality has a better iteration cycle and ownership when it's tested in rust. Can be done in a follow-up though

Reply:
Don't worry y'all I understand the benefit of testing closer to the real thing. The parsing tests in rust weren't easy for me to locate - I didn't properly find the rust parsing tests, so I thought that the bindings tests were the main ways you were testing specific functionality. There are no guidelines for how people can contribute to tests - I will update CONTRIBUTING.md to express this